fossunited/payments/doctype/razorpay_payment/razorpay_payment.py

```diff
@@ -83,3 +84,11 @@ def sync_status(self):
     @property
     def is_paid(self):
         return self.status == "Captured"
+
+    def before_insert(self):
+        if self.document_type == EVENT:
+            is_ticket_live = frappe.db.get_value(
+                EVENT, {"name": self.document_name}, "tickets_status"
```

Comment:
```suggestion
                EVENT, {"name": self.document_name}, "tickets_status"
```

Why is `tickets` plural here?

Reply:
Also a doesn't read like a name of a boolean/Check field

Reply:
Oh this is an error. `tickets_status` is a select. I am supposed to check if it is live in the if condition below

Reply:
Made changes for this